connect-inject/consul_sidecar_test.go

```diff
@@ -31,10 +31,12 @@ var (
 // that we pass the metrics flags to consul sidecar.
 func TestConsulSidecar_MetricsFlags(t *testing.T) {
```

Comment:
the var `consulSidecarResources` above here can be deleted since it's not used

Reply:
I also deleted the comment above since it was probably for different tests (when consul sidecar had other functionality)